.gitignore

```diff
@@ -3,4 +3,6 @@ node_modules/
 *.zip
 .idea/
 .DS_Store
-.vscode/
\ No newline at end of file
+.vscode/
+.env.*
+!.env.local
```

Comment:
if we ignore this, then we are forced to create a local config file?

Reply:
This prevents the .env.local from being ignored

Reply:
But yes, if we do that then we will have to create one ourselves